html2docx.py

```diff
@@ -1,155 +1,183 @@
-import io
-import re
+"""Parser for converting HTML to DOCX
+
+Accept input from a source like TinyMCE generated HTML. The
+source HTML may be preprocessed to limit tags and attributes.
+The input HTML must be valid with all closing tags (except
+for `br` tag).
+
+This module assumes a few conventions:
+
+- Block elements: `h1` to `h6`, `p`, `ol`, `ul`.
+- Any text inside a block element (`div`, `blockquote`, `pre`) that is
+  not inside a `p` element will have its own `p` element created
+  for it. The block element itself is not represented in the
+  resulting document.
+"""
+
 from html.parser import HTMLParser
-from typing import List, Tuple
+from io import BytesIO
```

Comment:
Why did the import change from `import io` to `from io import BytesIO`? For a large change like this, please avoid unrelated coding style changes. If there is a reason for the change, please make it a separate commit or PR with justification.